cyences_app_for_splunk/default/macros.conf

```diff
@@ -546,6 +549,31 @@ definition = eval main_event="1" \
 | fields - main_event
 iseval = 0
 
+[cs_current_week_login_count_o365]
+definition = eval main_event="1" \
+| append [ | tstats count, values(Authentication.org_country) as org_country from datamodel=Cyences_Authentication where Authentication.action="success" AND Authentication.user!="unknown" AND `cs_public_ips(Authentication.src)` AND `cs_o365_login_source` AND earliest="@w" latest="@d" by Authentication.app, Authentication.user, Authentication.src \
+| `drop_dm_object_name(Authentication)` \
+| iplocation src \
+| eval user=lower(user) \
+| eval Country = if(isnotnull(org_country), org_country, Country) \
+| inputlookup cs_authentication_usual_location.csv append=true \
+| eval user = lower(user) \
+| stats sum(count) as count by user, app, Country \
+| eventstats sum(count) as total by user, app \
+| eval percentage_login_from_country = round(count*100/total, 2) \
+| table user app Country percentage_login_from_country ] \
+| eventstats values(eval(Country." (". percentage_login_from_country. "%)")) as usual_login_location by user, app \
+| stats values(*) as * by user,app, Country \
+| search main_event="1" \
+| fields - main_event
+iseval = 0
+
+[cs_o365_login_source]
+definition = [|search `cs_o365` (sourcetype="o365:management:activity" OR source="AuditLogs.SignIns") earliest=-2h latest=now \
+| stats values(sourcetype) as sourcetype, dc(sourcetype) as count \
+| eval search=if(count==2 or (count==1 and sourcetype=="o365:graph:api"), "source=AuditLogs.SignIns", "sourcetype=o365:management:activity") \
```

Comment:
Shall we use search as 

sourcetype=o365:graph:api source=AuditLogs.SignIns

instead of just source?